strawberry/exceptions/handler.py

```diff
@@ -76,18 +68,18 @@ def strawberry_threading_exception_handler(
             # (we'd need to do type ignore for python 3.8 and above, but mypy
             # doesn't seem to be able to handle that and will complain in python 3.7)
 
-            cast(Any, original_threading_exception_hook)(args)
+            cast(Any, threading.__excepthook__)(args)
 
         return
 
     _get_handler(exception_type)(exception_type, exception, traceback)
 
 
 def reset_exception_handler():
-    sys.excepthook = original_exception_hook
+    sys.excepthook = sys.__excepthook__
 
     if sys.version_info >= (3, 8):
-        threading.excepthook = original_threading_exception_hook
+        threading.excepthook = threading.__excepthook__
```

Comment:
`threading.__excepthook__` is new in 3.10 not 3.8

Reply:
thanks! I'll revert it then!